src/lib/components/banner/banner.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import { closedBanners, close } from '$lib/stores/banner';
+  export let key: string;
+  export let severity: 'high' | 'medium' | 'low';
+  export let message: string;
+  export let link: string;
+
+  $: show = !(JSON.parse($closedBanners) ?? []).includes(key);
```

Comment:
There is a lot of logic to unpack here:

1. If it does _not_
2. include a key
3. from this parsed store
4. _or_ an empty array
5. show it

Reply:
It seems like we could move this business logic out of the view. Let some regular JavaScript function or derived store figure out what to show and just show the banners.

Reply:
wondering if this would be easier to read?
`$: hide = ($closedBanners ?? []).includes(key);`

about moving out the logic.. might go with the derived store, though what's the issue with having this in the component? seems pretty minimal